xlators/mgmt/glusterd/src/glusterd.c

```diff
@@ -698,18 +699,20 @@ runinit_gsyncd_setrx(runner_t *runner, glusterd_conf_t *conf)
     runner_add_arg(runner, "--config-set-rx");
 }
 
+static int
+run_gsyncd_cmd(runner_t *runner, xlator_t *this)
+{
+    int ret = 0;
+
+    ret = runner_run_reuse(runner);
+    if (ret == -1)
+        runner_log(runner, this->name, GF_LOG_ERROR, "command failed");
```

Comment:
Just a minor suggestion to make it more similar to our current style.
```suggestion
    if (ret < 0) {
        runner_log(runner, this->name, GF_LOG_ERROR, "command failed");
    }
```

Reply:
OK